x-pack/plugins/fleet/server/types/rest_spec/message_signing_service.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const RotateKeyPairSchema = {
+  query: schema.maybe(
```

Comment:
yeah, for a `POST` request, I would suggest using `body` and make it required that it must be `{ acknowledge: true` }